src/index.js

```diff
@@ -1,15 +1,13 @@
-import React from 'react'
+import React from "react";
 import ReactDOM from "react-dom/client";
-import { BrowserRouter } from 'react-router-dom';
 import "./index.css";
-import App from "../src/App";
+import App from "./App";
+import GlobalStyle from "./GlobalStyle";
 
 const root = ReactDOM.createRoot(document.getElementById("root"));
 root.render(
-  <React.StrictMode>
-    <BrowserRouter>
-      <App />
-    </BrowserRouter>
-  </React.StrictMode>
+  <>
+    <App />
+    <GlobalStyle />
```

Comment:
```suggestion
    <GlobalStyle />
    <App />
```

GlobalStyle은 적용하려고 하는 컴포넌트들의 가장 외각에 감싸주는 게 맞을 것 같아요!

Reply:
헉 글쿤요...적용이 안 되는데 해골물마냥 쓰고 있었을지도...🥲 저 Add suggestion to batch 신기하네용!!